core/arch/arm/plat-stm32mp1/drivers/stm32mp1_rcc.c

```diff
@@ -1,15 +1,64 @@
 // SPDX-License-Identifier: BSD-3-Clause
 /*
- * Copyright (c) 2018, STMicroelectronics
+ * Copyright (c) 2018-2019, STMicroelectronics
  */
 
 #include <drivers/stm32mp1_rcc.h>
+#include <io.h>
+#include <kernel/delay.h>
+#include <kernel/panic.h>
 #include <mm/core_memprot.h>
 #include <platform_config.h>
+#include <stm32_util.h>
+
+#define RESET_ID_MASK		GENMASK_32(31, 5)
+#define RESET_ID_SHIFT		5
+#define RESET_BIT_POS_MASK	GENMASK_32(4, 0)
+
+#define RST_CLR_OFFSET		4U
```

Comment:
I guess this means that every second word is a "reset clear" register. A comment giving some light overview of the layout would be welcome.

Reply:
Ok, thanks for the feedback.